runtime/include/chpl-topo.h

```diff
@@ -50,6 +50,10 @@ void* chpl_topo_getHwlocTopology(void);
 int chpl_topo_getNumCPUsPhysical(chpl_bool /*accessible_only*/);
 int chpl_topo_getNumCPUsLogical(chpl_bool /*accessible_only*/);
 
+// Which CPUs (cores/PUs) are accessible?
+
+int chpl_topo_getCPUs(chpl_bool physical, int *cpus, int size);
```

Comment:
Can you describe the interface a little more? Seems like will fill in an array of cpu indexes with the index value if it's accessible and then return the total number of cores? Does the value match `chpl_topo_getNumCPUs*`?